src/app/pages/ArticlePage/ArticlePage.tsx

```diff
@@ -176,6 +215,25 @@ const ArticlePage = ({ pageData }: { pageData: Article }) => {
 
   const isInServerSideExperiment =
     experimentVariant && experimentVariant !== 'off';
+
+  // EXPERIMENT: Read Time
+  const readTimeExperimentName = 'newswb_ws_article_read_time';
+  const readTimeExperimentVariant = useOptimizelyVariation({
+    experimentName: readTimeExperimentName,
+    experimentType: ExperimentType.CLIENT_SIDE,
+  });
```

Comment:
Doing this here isn't too great as this means we're going to be requesting the datafile on all services for articles. Would be better if the variation was decided at ReadTime component level, but we need to know the variation to do the split rendering between the timestamp and headline

Reply:
Scrap the above this would happen anyway, as metrics will make use of the datafile